operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
 
 ### Document sections
 
-To make pages easier for people to quickly scan for the content that they're looking for, break your 
-document up into logical sections. Each section should have a heading, and the heading should relate 
-to the content of the section. A section heading is not required if you have only one section.
+To make pages easier for people to quickly scan for the content that they're looking for, break your document up into logical sections. Each section should have a heading, and the heading should relate to the content of the section. A section heading is not required if you have only one section.
 
 **Use sentence case for the section heading (e.g. "This section is about types of headings") except where a word/phrase is a proper noun (e.g. "Mattermost"), the name of a country/region (e.g. "The United States of America"), and so on.**
 
-
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
```

Comment:
I wonder if we can make this a bolded subsection: **Customer Education** so it's easier to see the transition? 3/5 for easier reading.

Reply:
I've thought about it, but then I thought it would be inconsistent with all the other CE subsection of the other topics.

How about changing

> An exception is Customer Education content that [...]

to

> An exception is **Customer Education** content that [...]

?

Reply:
That works for me. :)